<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Text</class>
 <widget class="QDialog" name="Text">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="Form">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Font Color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="colorBox">
       <property name="text">
        <string>#000000</string>
       </property>
       <property name="maxLength">
        <number>7</number>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Font Family</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Alignment</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="alignBox">
       <item>
        <property name="text">
         <string>Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Center</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Right</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Justified</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="fontFaceBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="PlainText">
     <property name="cursor" stdset="0">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Insert your text here</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="OkButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="CancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
